packages/utils/tool-utils/package.json

```diff
@@ -65,6 +65,7 @@
 	},
 	"dependencies": {
 		"@fluidframework/common-utils": "^1.1.1",
+		"@fluidframework/driver-utils": "workspace:~",
 		"@fluidframework/odsp-doclib-utils": "workspace:~",
 		"@fluidframework/protocol-base": "^0.1039.1000",
```

Comment:
is protocol-base still needed in all these packages?

Reply:
i would also expect code changes to reference driver utils, otherwise where is no point in adding the dependency

Reply:
protocol-base will be still required in driver-utils as we have `buildHierarchy` function imported:https://github.com/microsoft/FluidFramework/blob/main/packages/loader/driver-utils/src/buildSnapshotTree.ts#L14 which is not yet moved and is used by both client and server. But I'll remove the dependency from other places.